### PR TITLE
Wrap OrtGraph API into OrtModelBuilder

### PR DESCRIPTION
**Pull request by shiyi9801**

### CODE REVIEW

**Review by huningxin — APPROVED** [review]
LGTM!

**Comment by huningxin** [comment]
@mingmingtasd, please also take a look.

File: services/webnn/ort/scoped_ort_types.h [hunk 1]
```diff
@@ -97,7 +111,9 @@ class ScopedOrtNode {
 
 class ScopedOrtModel {
  public:
-  ScopedOrtModel();
+  explicit ScopedOrtModel();
```
Comment (mingmingtasd) [inline services/webnn/ort/scoped_ort_types.h:114]: It seems these  `explicit` are unnecessary, `ScopedOrt**` constructor has no parameters.

Comment (shiyi9801) [reply services/webnn/ort/scoped_ort_types.h]: 👍 Removed

**Comment by mingmingtasd** [comment]
LGTM!

**Pull request merged**